Packs/CSCDomainManager/Integrations/CSCDomainManager/CSCDomainManager.py

```diff
@@ -0,0 +1,553 @@
+        A list of available domains with the fields for human readable
+    """
+    filtered_available_domains = []
+
+    for domain in available_domains:
+        filtered = {
+            'Qualified Domain Name': domain.get('qualifiedDomainName'),
+            'Code': domain.get('result').get('code'),
+            'Message': domain.get('result').get('message'),
+            'Price': domain.get('basePrice').get('price'),
+            'Currency': domain.get('basePrice').get('currency'),
+            'List of the terms (months) available for registration': domain.get('availableTerms')
+        }
+
+        filtered_available_domains.append(filtered)
+
+    return filtered_available_domains
+
+
+def get_domain_hr_fields(domain) -> dict:
+    """
+    Create a dict of the domain with the fields for human readable, using the domain argument
+
+    Args:
+        domain: domain dict that was output from the http request
+
+    Returns:
+        A dict of the domain with the fields for human readable
+    """
+    filtered = {'Qualified Domain Name': domain.get('qualifiedDomainName'),
+                'Domain': domain.get('domain'),
+                'Idn': domain.get('idn'),
+                'Generic top-level domains': domain.get('newGtld'),
+                'Managed Status': domain.get('managedStatus'),
+                'Registration Date': domain.get('registrationDate'),
+                'Registry Expiry Date': domain.get('registryExpiryDate'),
+                'Paid Through Date': domain.get('paidThroughDate'),
+                'Country Code': domain.get('countryCode'),
+                'Server Delete Prohibited': domain.get('countryCode'),
+                'Server Transfer Prohibited': domain.get('serverDeleteProhibited'),
+                'Server Update Prohibite d': domain.get('serverTransferProhibited'),
+                'Name Servers': domain.get('nameServers'),
+                'Dns Type': domain.get('dnsType'),
+                'Whois Contact first Name': domain.get('whoisContacts')[0].get('firstName'),
+                'Whois Contact last Name': domain.get('whoisContacts')[0].get('lastName'),
+                'Whois Contact email': domain.get('whoisContacts')[0].get('email')
+                }
+
+    domain.get('whoisContacts')
+    return filtered
+
+
+def get_whois_contacts_fields_for_search_domains(whois_contacts, field_name: str) -> list:
+    """
+    Create a list of contact.field_name for each contact in whois_contacts. Specific arrangement for the search domains command
+
+    Args:
+        whois_contacts: list of contacts
+        field_name: the field to get for each contact
+
+    Returns:
+        A list of contact.field_name
+    """
+    results = []
+    for contact in whois_contacts:
+        results.append(contact.get(field_name))
+
+    return results
+
+
+def get_whois_contacts_fields_for_domain(whois_contact, field_names: str | List[str],
+                                         contact_type_condition: str) -> list:
```

Comment:
```suggestion
def get_whois_contacts_fields_for_domain(whois_contact, contact_type_condition: str, *field_names: List[str]) -> list:
```
use varargs and always recieve a list.